dluhc-component-library/src/components/timetablePage/Timetable.tsx

```diff
@@ -74,11 +75,17 @@ const Timetable = ({ data }: StagesProps) => {
                 ))}
               </tr>
               <tr className="border-b">
-                <div className="pt-1 pb-3">
-                  <span className="text-blue-400 underline">
-                    More information
-                  </span>
-                </div>
+                <td colSpan={4}>
+                  <AccordionDropdown
+                    information="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
Not for this PR but could we prioritise getting this information from the json / csv file please as its known how we will get it unlike the other hardcoded values.